build.xml

```diff
@@ -4,11 +4,31 @@
     <property name="path.root" value="${project.basedir}/project-base"/>
     <property name="path.vendor" value="${project.basedir}/vendor"/>
     <property name="path.packages" value="${project.basedir}/packages"/>
+    <property name="path.backend-api" value="${path.packages}/backend-api"/>
     <property name="path.framework" value="${path.packages}/framework"/>
     <property name="path.utils" value="${project.basedir}/utils"/>
     <property name="translations.dump.flags" value=""/>
 
     <import file="${path.framework}/build.xml"/>
+    <import file="${path.backend-api}/build.xml"/>
+
+    <target name="build-demo" depends="monorepo-backend-api-install,composer-prod,backend-api-oauth-keys-generate,shopsys_framework.build-demo,monorepo-backend-api-uninstall" description="Builds application for production with clean demo DB."/>
+
+    <target name="build-demo-ci" depends="monorepo-backend-api-install,composer-prod,backend-api-oauth-keys-generate,shopsys_framework.build-demo-ci,monorepo-backend-api-uninstall" description="Builds application for development with clean demo DB and runs CI checks."/>
```

Comment:
```suggestion
    <target name="build-demo-ci" depends="monorepo-backend-api-install,composer-dev,backend-api-oauth-keys-generate,shopsys_framework.build-demo-ci,monorepo-backend-api-uninstall" description="Builds application for development with clean demo DB and runs CI checks."/>
```
you need to test it

Reply:
`build-demo` uses `composer-prod`, why do you suggest `-dev`?

Reply:
ci is for CI so you need to have to run tests and those are in dev dependencies